projects/dotcms-field-elements/src/components/dot-form/dot-form.scss

```diff
@@ -2,7 +2,7 @@ dot-form {
     display: block;
 
     & > form {
-        & > .form__fields > * {
+        .form__fields > * {
```

Comment:
You sure about this?

Reply:
yes, I think this was not being applied before